src/main/java/org/moonlightcontroller/managers/models/messages/ProcessingGraphMessage.java

```diff
@@ -0,0 +1,61 @@
+package org.moonlightcontroller.managers.models.messages;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.openboxprotocol.protocol.IStatement;
+
+public class ProcessingGraphMessage implements IMessage {

```

Comment:
Does this class correspond to the SetProcessingGraphRequest message in the spec?
If so, the class should be named SetProcessingGraphRequest (and in general, all message classes should be named after the protocol message they represent)